README.md

```diff
@@ -0,0 +1,41 @@
+# ExchequerClient
```

Comment:
kill the boiler plate here please

Reply:
just this top section though, maybe put that same description in again too, and link to the server github page.